hack/local-up-cluster.sh

```diff
@@ -554,7 +540,7 @@ EOF
     fi
 
     APISERVER_LOG=${LOG_DIR}/kube-apiserver.log
-    ${CONTROLPLANE_SUDO} "${GO_OUT}/hyperkube" kube-apiserver "${authorizer_arg}" "${priv_arg}" ${runtime_config} \
+    ${CONTROLPLANE_SUDO} "${GO_OUT}/hyperkube" kube-apiserver "${authorizer_arg}" "${priv_arg}" "${runtime_config}" \
```

Comment:
drop this change

Reply:
It's that "shell-hack" script complaining about it without quotes.

Reply:
if the var is empty, "" will be incorrect

Reply:
I think `""` won't be dumped - if the var is empty, `"${runtime_config}"` still represents an empty string instead of `""`.

```bash
$ cat /tmp/test.sh
#!/bin/bash
runtime_config=""
echo X${runtime_config}Y
echo X"${runtime_config}"Y

$ /tmp/test.sh
XY
XY
```

Reply:
"${runtime_config}" would show up as an empty arg to hyperkube
${runtime_config} would not cause additional args

Reply:
ok, I changed it back and added an explicit shell-check exception.